metacat-main/src/test/groovy/com/netflix/metacat/main/services/impl/TableServiceImplSpec.groovy

```diff
@@ -463,20 +473,48 @@ class TableServiceImplSpec extends Specification {
         1 * config.isParentChildRenameEnabled() >> false
         1 * parentChildRelSvc.isChildTable(oldName) >> false
         1 * parentChildRelSvc.isParentTable(oldName) >> false
-        1 * parentChildRelSvc.rename(oldName, newName)
+        1 * parentChildRelSvc.renameSoftInsert(oldName, newName) >> emptyPair
+        1 * connectorTableServiceProxy.rename(oldName, newName, _)
+        0 * parentChildRelSvc.drop(oldName, _)
+        noExceptionThrown()
+
+        // mock normal success case that is not a parent nor child table
+        when:
+        service.rename(oldName, newName, false)
+        then:
+        1 * config.getNoTableRenameOnTags() >> []
+        1 * config.isParentChildRenameEnabled() >> true
+        0 * parentChildRelSvc.isChildTable(oldName)
+        0 * parentChildRelSvc.isParentTable(oldName)
+        1 * parentChildRelSvc.renameSoftInsert(oldName, newName) >> emptyPair
+        1 * connectorTableServiceProxy.rename(oldName, newName, _)
+        0 * parentChildRelSvc.drop(oldName, Optional.of(emptyPair))
```

Comment:
```suggestion
        0 * parentChildRelSvc.drop(_, _)
```

nit: just more comprehensive sicne we dont expect any dropping when the pair was just empty sets